extension/src/test/suite/plots/index.test.ts

```diff
@@ -1124,6 +1124,23 @@ suite('Plots Test Suite', () => {
       )
     })
 
+    it('should handle a add template plot mmessage from the webview', async () => {
+      const { mockMessageReceived } = await buildPlotsWebview({
+        disposer: disposable,
+        plotsDiff: plotsDiffFixture
+      })
+
+      const executeCommandSpy = spy(commands, 'executeCommand')
+
+      mockMessageReceived.fire({
+        type: MessageFromWebviewType.ADD_PIPELINE_PLOT
+      })
+
+      expect(executeCommandSpy).to.be.calledWithExactly(
+        RegisteredCommands.PIPELINE_ADD_PLOT
```

Comment:
[B] We know which `dvcRoot` we are calling `RegisteredCommands.PIPELINE_ADD_PLOT` from so we should not use `getCwd` again. Otherwise when a user is using a mono-repo they will click on "Add Plot" from a specific repos plots' webview and then be asked to choose which repository they want to add the plot to. There are plenty of examples of how we normally do this in `extension/src/experiments/webview/messages.ts` (e.g. `RegisteredCliCommands.EXPERIMENT_VIEW_PUSH`)

Reply:
Good catch! Updating the command to check for a `dvcRoot` being passed.